sklearn/naive_bayes.py

```diff
@@ -285,9 +310,14 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
 
         classes : array-like, shape (n_classes,), optional (default=None)
             List of all the classes that can possibly appear in the y vector.
+            If classes argument was set in initialization, then this will be
+            ignored.
 
-            Must be provided at the first call to partial_fit, can be omitted
-            in subsequent calls.
+            Must be provided at the first call to partial_fit or during class
+            initialization. It will be ignored in subsequent calls.
+
+            Note: Now this argument is redundant. We should remove this. But
+            this will affect backward compatibility
```

Comment:
is removing classes argument from partial_fit feasible? it will cause the previous code using this argument to give an error. thoughts?